### PR TITLE
Consistency for README.md examples.

### PR DESCRIPTION
**Pull request by shockpast**
Optional and questionable, didn't actually found out what is your **main** typing formatting.

Answer me, what is better formatting from your view, i'm edited to make more consistent, because sometimes you did spaces between brackets, and then you didn't, it's just looks strange and ugly.
```
chatBubbleScale( random(0.2, 2.0) ) 
- with spaces between brackets.
```
```
chatBubbleScale(random(0.2, 2.0))
- without spaces between brackets.
```

### CODE REVIEW

**Comment by shockpast** [comment]
Also removed `_` in the hook `'PlayerSay'`, because it didn't used, so you can just don't name it, imperceptibly, but more convenient.

**Comment by StyledStrike** [comment]
To be honest, these are mistakes were made because of _when_ I was writing these examples, at the same time as the rest of the code, in a point where making some things easier to read while testing were the priority.

At some point I really need to go over my repos and apply consistency rules to the code... Oh actually [CFC has good guidelines](https://github.com/CFC-Servers/cfc_glua_style_guidelines). I may as well follow them soon.

In short, code formatting wasn't "too important" for me back then but I certainly should make them be.
Thanks for the PR by the way!
